HHS/Head-Start-TTADP
# [TTAHUB-1615] add what's new notifications from confluence

## Description of change
Fetch content from Confluence to populate the "notifications" menu item for users.

![Screenshot 2023-03-29 at 10 34 39 AM](https://user-images.githubusercontent.com/3288586/228573081-8cc0e8af-9641-454c-b79a-4ac86541fbb5.png)

Uses local storage to persist read or unread feed information.
Data is fetched for the previous 90 days for a maximum of 25 posts. 

You'll see assorted changes that seem unrelated but were made as I tried to get to get this through the CI pipeline.

## How to test
Validate that the data is fetched, that the "unread" indicators are accurate, and that the display of the data matches the mockup. Note that we are somewhat at the mercy of what comes back from Confluence, so there might be additional spaces after the content that doesn't exactly match the mockup. You'll see what I mean.

Another place where the implementation differs from the mockup is the "back" button on the Notifications page. I am tracking the referrer but we are not in the business of storing the previous page's title (nor, philosophically, do I feel like we need to completely recreate the browser's back button's functionality in our app). So it just says "back" and not "back to whatever."

## Issue(s)
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1615

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
